apps/vs-code-designer/src/README.md

```diff
@@ -0,0 +1,74 @@
+# Azure Logic Apps for Visual Studio Code (Standard)
+
+Use the Azure Logic Apps (Standard) extension to quickly create, debug, manage, and deploy Logic Apps directly from VS Code.
+
+> Sign up today for your free Azure account and receive 12 months of free popular services, $200 free credit and 25+ always free services 👉 [Start Free](https://azure.microsoft.com/free/open-source).
+
+## Create your first logic app
```

Comment:
@ecfan The reason we get the steps in here is because this README file gets rendered in the visual studio marketplace https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurelogicapps

I get the idea of your comment about outdated steps. Should I remove the steps in here and provide a link instead or keep the steps and add the link?

Should I add an `aka` link? or what is a FW link?

Reply:
I'll provide an FW link instead. This link type is specifically set up for in-product redirects, whereas aka.ms links are for short URLs used for marketing purposes and are usually short-lived. For more info, see [FW links versus aka.ms links](https://review.learn.microsoft.com/en-us/help/contribute/contribute-link-to-articles-from-the-user-interface?branch=main#fwlinks-and-akams-links).

Reply:
Awesome! Thank you very much for the info link